src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -113,6 +143,7 @@ const ChatItemMyTextV2 = ({
   time = '4h',
   isContinuous = false,
   message = '',
+  attachments = [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new prop `attachments` has been added to the component. Make sure this prop is always provided when using this component, or handle the case when it's undefined or null to prevent potential errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->